tests/conftest.py

```diff
@@ -523,6 +523,9 @@ def exc_handle(loop, context):
 
     yield hass
 
+    for config_entry in hass.config_entries.async_entries():
+        loop.run_until_complete(config_entry.async_unload(hass))
```

Comment:
Should we unload them in a gather?

Reply:
Done

Reply:
I had to revert it:
`RuntimeError: Task got bad yield: <coroutine object ConfigEntry.async_unload at 0x7f2f09aa0ba0>`

Reply:
I think if we want to use `gather` we need to make the whole `hass` fixture async.

I am not sure why it is not marked as async currently, but I figure there might be a reason for that.

I think we can always migrate to async in a follow-up PR.

Reply:
I've opened #91264 to migrate the `hass` fixture.
I will adjust this PR accordingly if it is merged first.

Reply:
#91264 has been merged.
I have adjusted this PR accordingly.

Reply:
Aargh - going async has uncovered extra issues